### PR TITLE
curryst:0.1.1

### PR DESCRIPTION
**Pull request by pauladam94**
- Adding Examples
- Fixing typesetting issue for rule with only one name

### CODE REVIEW

**Comment by MDLC01** [comment]
Wasn't the previous version already 0.1.0?

**Comment by emilyyyylime** [comment]
@MDLC01 The previous version wasn't yet accepted

**Comment by MDLC01** [comment]
Isn't this the previous version : https://github.com/typst/packages/tree/main/packages/preview/curryst/0.1.0
Which was added in the last commit [`c0cf9fc`](https://github.com/typst/packages/commit/c0cf9fcbf9c0626e5c0f385889e528c49784de86)

**Comment by laurmaedje** [comment]
The previous version was indeed accepted, so these changes would have to be submitted as a new version.

**Comment by pauladam94** [comment]
My bad i should have change the version. I will do it.

**Comment by laurmaedje** [comment]
Hey @pauladam94, is this PR still on your radar?

**Comment by pauladam94** [comment]
> Hey @pauladam94, is this PR still on your radar?

Yes. I am just very busy and I have not take the time now to complete this PR.
I will have time in the next two weeks approximately.

**Comment by pauladam94** [comment]
@laurmaedje I think everything is in order now.

File: packages/preview/curryst/0.1.1/README.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
+# Curryst
+
+A Typst package for typesetting proof trees. You can import this package with :
+
+```typst
+#import "@preview/curryst:0.1.0"
```
Comment (laurmaedje) [inline packages/preview/curryst/0.1.1/README.md:6]:
```suggestion
#import "@preview/curryst:0.1.1"
```

**Review by pauladam94 — COMMENTED** [review]
version fix in the example on how to import the package

**Comment by pauladam94** [comment]
Do I have to do something else about this PR ?

**Comment by laurmaedje** [comment]
It looks like you didn't push the change.

**Comment by pauladam94** [comment]
@laurmaedje I think I have now ? I am not sure what has to be done on my part.

**Comment by laurmaedje** [comment]
The readme still says "0.1.0" in GitHub rather than "0.1.1":

**Comment by pauladam94** [comment]
Ok I thought that your comment on the code had an effect. But I had to change it my self ! It's done now I think.

**Comment by laurmaedje** [comment]
Perfect, thanks!

**Pull request merged**